ambry-cloud/src/test/java/com/github/ambry/cloud/LatchBasedInMemoryCloudDestination.java

```diff
@@ -70,6 +72,11 @@
   private final Set<CosmosContainerDeletionEntry> deprecatedContainers = new HashSet<>();
   private final ClusterMap clusterMap;
 
+  // Used in error simulation.
```

Comment:
I think this implementation should be the only place we expose those error injection methods.

Reply:
Modified.